drivers/sony/displays/bravia.cr

```diff
@@ -0,0 +1,213 @@
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/switchable"
+
+class Sony::Displays::Bravia < PlaceOS::Driver
+  include Interface::Powerable
+  include Interface::Muteable
+
+  INDICATOR = "\x2A\x53" # *S
+  msg_length = 21
+
+  INPUTS = {
```

Comment:
We would want to use the Switchable interface inputs, this is how it's done for a [Samsung driver](https://github.com/PlaceOS/drivers/blob/master/drivers/samsung/displays/mdc_protocol.cr#L11-L31)